<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vdoplayer</class>
 <widget class="QWidget" name="vdoplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/resources/background.jpg);</string>
  </property>
  <widget class="QPushButton" name="BtnBackHome">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenFileBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PlayBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StopBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FullScrBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>全屏</string>
   </property>
  </widget>
  <widget class="QSlider" name="ProgressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="wid_vdo" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>711</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources/background.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabRatio">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>00.00:00.00</string>
   </property>
  </widget>
  <widget class="QSlider" name="SliderVolumn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="SoundBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
